test/cli/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tutorialkit-integration",
+  "name": "@tutorialkit-cli/test-cli",
```

Comment:
I would keep the previous name and the `integrations` folder name TBH. 

I prefer reserving `test` (or `tests`) for unit tests.

Let's move the ui tests in a `e2e` folder at the root of the project.

Reply:
I thought having just a single folder in the root would be more organized. We have similar structure here: https://github.com/vitest-dev/vitest

But sure, I'll revert these and create a `<root>/e2e` instead.

Reply:
In WebContainer we have the following conventions:

 - `*.spec.ts` files in either `test` or in `src` folders are unit tests
 - `integrations` contains tests that verify the logic at the node level
 -  `e2e` contains tests that look at the entire webcontainer infra treating it as a black box (more or less)

I think what we're doing with our UI tests here is akin to e2e. We're testing the product in its entirety.

I find it nice to follow similar conventions here because:
 - That was the plan when TutorialKit was created.
 - Having similar conventions between projects makes context switching easier.

However, happy to discuss it with the rest of the team and decide with them if we want to do something different  for TutorialKit.